packages/syft/src/syft/client/client.py

```diff
@@ -527,6 +528,27 @@ def create_project(
         project = project_create.start()
         return project
 
+    def sync_code_from_request(self, request):
+        code = deepcopy(request.code)
```

Comment:
[Nitpick] Can we also check if request.code is a UserCode obj and not None or SyftError